src/Jobs/QueueableJob.php

```diff
@@ -11,6 +11,8 @@ class QueueableJob implements ShouldQueue
     use Queueable;
     use ProgressReportable;
 
+    public int $timeout = 60;
```

Comment:
This should probably be nullable and `null` by default. See my more detailed comment.